src/templates/default_function/app.py

```diff
@@ -3,8 +3,8 @@
 import sys
 
 
-def hello() -> str:
-    return "Hello World!"
+def hello(name: str) -> str:
```

Comment:
If we need to change template of function files, then we have to adjust integration tests.

Please run:
```
pytest -m integration tests_integration/test_snowpark_function.py
```